zhaquirks/philips/rdm002.py

```diff
@@ -69,6 +75,48 @@ class PhilipsRdm002RemoteCluster(PhilipsRemoteCluster):
     }
 
 
+class PhilipsRdm002LevelControl(LevelControl):
```

Comment:
```suggestion
class PhilipsRdm002LevelControl(CustomCluster, LevelControl):
```
(and import of `CustomCluster` should make it work)